tests/sysprep_test.go

```diff
@@ -45,11 +36,18 @@ import (
 
 	v1 "kubevirt.io/api/core/v1"
 	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/pkgs/libvmi"
```

Comment:
```suggestion
	"kubevirt.io/kubevirt/pkg/libvmi"
```

Reply:
thanks and sorry for the noise. I should have left this as draft.